Jenkinsfile

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/groovy
+library 'adminsLib@pull/1/head'
+import groovy.json.JsonOutput
+
+properties([
+    parameters([
+        string(defaultValue: '', description: 'deb-drop repository, see https://github.com/innogames/deb-drop/', name: 'REPO_NAME', trim: false),
+        booleanParam(defaultValue: false, description: 'Publish new packages and images or not. True for new releases, False by default.', name: 'PUBLISH'),
+    ])
+])
+
+def build_packages(docker_image) {
+    docker_image.inside("${jobCommon.dockerArgs()} -e GOPATH='${env.HOME}/go'") {
+        sh '''\
+            #!/bin/sh -ex
+            make packages
+            '''.stripIndent()
+    }
+}
+
+
+// Remove builds in presented status, default is ['ABORTED', 'NOT_BUILT']
+jobCommon.cleanNotFinishedBuilds()
+
+node('docker') {
+ansiColor('xterm') {
+    // Checkout repo and get info about current stage
+    sh 'echo Initial env; env | sort'
+    env.PACKAGE_NAME = 'graphite-ch-optimizer'
+    env.DOCKER_IMAGE = 'innogames/' + env.PACKAGE_NAME
+    def img_builder
+    def img_build
+    try {
+        stage('Checkout') {
+            gitSteps checkout: true, changeBuildName: false
+            sh 'set +x; echo "Environment variables after checkout:"; env|sort'
+            env.NEW_VERSION = sh(returnStdout: true, script: 'make version').trim()
+            currentBuild.displayName = "${currentBuild.number}: version ${env.NEW_VERSION}"
+        }
+        stage('Tests') {
+            try {
+                docker.image("${env.DOCKER_IMAGE}:builder").pull()
+            } catch (all) {
+                echo 'Unable to pull builder image, building from scratch'
+            } finally {
+                img_builder = docker.build(
+                        "${env.DOCKER_IMAGE}:builder",
+                        "--target builder --cache-from=${env.DOCKER_IMAGE}:builder ."
+                        )
+            }
+            docker.withRegistry('', 'dockerIgSysadminsToken') {
+                img_builder.push()
+            }
+            img_build = docker.build("${env.DOCKER_IMAGE}:build", '--target build .')
+            img_build.inside("${jobCommon.dockerArgs()} -e GOPATH='${env.HOME}/go'") {
+                sh '''\
+                    #!/bin/sh -ex
+                    make test
+                    go get -u
+                    if ! git diff --exit-code; then
```

Comment:
i don't understand why you are doing this. why go get `-u`? don't do that. and then remove the `git diff` command

Reply:
removed, as discussed that update dependencies is a maintainers' task